l10n/en/mozorg/home-m24.ftl

```diff
@@ -13,6 +13,8 @@ m24-home-page-desc = Did you know? { -brand-name-mozilla } — the maker of { -b
 
 ## Intro
 
+m24-home-pause-animation = Pause animation
```

Comment:
question for reviewer: should these be in a more generic UI ftl file?

Reply:
Good idea. That's a global file so the string would be available everywhere.